release-notes/rc3-download.md

```diff
@@ -0,0 +1,72 @@
+# .NET Core SDK 1.0 RC3 build 004517
+
+The installers and binary archives on this page include .NET Core 1.0 SDK RC 3.
+
+| .NET Core 1.0 Preview 4 | Installer                                        | Binaries                                        |
```

Comment:
The page is about RC3, but this line talks about Preview 4. I think that should be fixed.